### ISSUE TITLE
Grid View?

### ISSUE THREAD

**Issue by volomike** [anuraghazra/BugVilla#21]
Any chance on a grid view?

**Comment by anuraghazra** [comment]
Yes a grid view would be nice. Probably a toggle button for different views maybe, like list, grid & masonry. 

I'll add this. Meanwhile you can also try to implement it. 😃 contributions are welcomed🎉

**Comment by volomike** [comment]
> Meanwhile you can also try to implement it. 😃 contributions are welcomed🎉

Already on it. I'm working on a PHP-based bug tracker that I intend to make as a SaaS app. Right now I'm working on the UX ideas as well as learning things like super-fast subdomain creation (google "wildcard subdomains"), the file storage issue (Wasabi versus gambling with shared hosting space predictions), and command line provisioning (to move customer file storage to other hosts super fast without them knowing, as they grow).

**Comment by volomike** [comment]
Meanwhile, you taught me a new term -- masonry view. https://masonry.desandro.com/ I didn't know it was called that.
